oojs.md

```diff
@@ -8,29 +8,55 @@ Define a `Playlist` class. It should have...
 
 ```js
 Your answer goes here
-```
+<!-- Beginning of answer. -->
+class playList {
+		constructor(title, song){
+		this.title = title;
+		this.song = [];
+		this.favorites = prompt("Add rating of favorites from 0-5 here.");
+	}
+	addSong(){
+		this.song = prompt("Add song of choice to current play list you are creating.")
+	}
+}
+<!-- End of my answer here. -->
 
 ### 2
 
 Create an instance of the `Playlist` class.
 
 ```js
 Your answer goes here
-```
-
+<!-- Beginning of answer here. -->
+const list1 = new playList("Burning Bridges", "American Love");
+	console.log(list1.title)
+	console.log(list1.song)
+	list1.addSong();
+<!-- End of answer here. -->
 ### 3
 
 Create an `Album` class that inherits from `Playlist`. It should also...
 - Have an `artist` (String) property that is determined by some input
 
 ```js
 Your answer goes here
-```
+<!-- Beginning of answer here. -->
+class Album extends playList {
+	constructor(artist){
```

Comment:
nice. You'll have to call super, though too, and take all of the things in this constructor and pass them to the parent constructor